### ISSUE TITLE
Miscellaneous bugs and tasks

### ISSUE THREAD

**Issue by jasloe** [aarome/aarome#109]
Support images not rendered
Extra arrow on /about section front
Place links, PDF, call-to-action below paragraphs
Underline links in page sections/paragraphs
Enable breadcrumbs
Enable /news, /features inside pages

**Comment by jasloe** [comment]
@cchristopherhowardd: cleaned up support. What do you want to do with the Fall Gala and McKim Gala blocks?

**Comment by jasloe** [comment]
@cchristopherhowardd: removed extra arrow from /about.

**Comment by jasloe** [comment]
Paragraph/page section links, refer to: http://dev.aarome.org/about/mission

**Comment by jasloe** [comment]
Refer to this Twig snippet for placing links, PDFs, and calls-to-action in the correct order and position: https://gist.github.com/jasloe/01ce95bd17d4c7fccb0043059f40b24b

**Comment by cchristopherhowardd** [comment]
Fall Gala and McKim Gala should be a single block: Galas. URL can be aarome.org/support/galas.

The Society of Fellows block in the Support section should be removed entirely.

**Comment by cchristopherhowardd** [comment]
Another change is needed for About. The second block (Academy) should be named Place. The link should go to aarome.org/about/place.

**Comment by jasloe** [comment]
> Another change is needed for About. The second block (Academy) should be named Place. The link should go to aarome.org/about/place.

Fixed. Tidying up the page sections now.

**Comment by cchristopherhowardd** [comment]
At aarome.org/about, I'd still like the section/header/word "Academy" replaced by "Place." At aarome.org/about/place, however, "Academy" can stay.

**Comment by jasloe** [comment]
> At aarome.org/about, I'd still like the section/header/word "Academy" replaced by "Place." At aarome.org/about/place, however, "Academy" can stay.

Right, that's just a matter of changing the content title. I went ahead and did that.

**Comment by cchristopherhowardd** [comment]
But changing one changes both! I need the word at /about to be Place, and at about/place to be Academy.

**Comment by jasloe** [comment]
Huh. Try refreshing /about a couple of times. I overrode the view query since both views use the same node. (I'll rebuild the view with a fresh node when I get a chance.)
